crates/backend/Cargo.toml

```diff
@@ -8,4 +8,6 @@ readme = "README.md"
 
 [dependencies]
 actix-web = "3.3.2"
-actix-files = "0.5.0"
\ No newline at end of file
+actix-files = "0.5.0"
+envy = "0.4"
```

Comment:
As already discussed, we obtain strongly typed .env variables but add two extra dependencies. However, AFAIK this is not a performance issue on a Rust backend.